frontend/app/services/MembersDataAPI.scala

```diff
@@ -85,36 +85,31 @@ class MembersDataAPI(executionContext: ExecutionContext) {
             val prefix = s"members-data-api-check identity=${memberRequest.user.id} salesforce=${memberRequest.subscriber.contact.salesforceContactId} : "
             val salesforceAttrs = Attributes.fromMember(memberRequest.subscriber)
             if (memDataApiAttrs != salesforceAttrs) {
-              val message = s"$prefix MISMATCH salesforce=$salesforceAttrs mem-data-api=$memDataApiAttrs"
-              if (memDataApiAttrs.tier != salesforceAttrs.tier) Logger.error(message) else Logger.warn(message)
-              MembersDataAPIMetrics.put("members-data-api-mismatch", 1)
-            } else {
-              Logger.debug(s"$prefix MATCH")
-              MembersDataAPIMetrics.put("members-data-api-match", 1)
+              SafeLogger.error(scrub"$prefix MISMATCH salesforce=$salesforceAttrs mem-data-api=$memDataApiAttrs")
```

Comment:
I don't think there's any good reason why these attributes shouldn't match any more, but I noticed that these errors still crop up from time to time. Consequently I'd like to keep a simplified version of this logging in for now...